R/objects.R

```diff
@@ -25,6 +25,60 @@ CellGraphs.Seurat <- function (
   return(CellGraphs(object = object[[assay]]))
 }
 
+#' @method FSMap Seurat
+#'
+#' @rdname FSMap
+#'
+#' @examples
+#' pxl_file <- system.file("extdata/five_cells",
+#'                         "five_cells.pxl",
+#'                         package = "pixelatorR")
+#' seur_obj <- ReadMPX_Seurat(pxl_file)
+#'
+#' # Check PXL file paths in a Seurat object
+#' FSMap(seur_obj)
+#'
+#' @export
+#'
+FSMap.Seurat <- function (
+  object,
+  ...
+) {
+  assay <- DefaultAssay(object = object)
```

Comment:
Is there any occasion one would want to check the FSmap on a non-default assay?

Reply:
Yeah I was also thinking about that. We could add that as an option!

I wouldn't expect that to be very useful for beginners, but maybe for more advanced users?

Reply:
I agree that it would probably be an edge case. I am fine with either or!